backend/Dockerfile.visualization

```diff
@@ -17,7 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM tensorflow/tensorflow:2.1.0-py3
+FROM tensorflow/tensorflow:2.2.0
```

Comment:
just realized you should also update: https://github.com/kubeflow/pipelines/blob/0b096de63faaf06c38446e0b053b87f7c51f3925/backend/src/apiserver/visualization/update_requirements.sh#L4

and Can you add a comment here reminding people about that

Reply:
Done.
Although it's not strictly required that these images are in sync. It just makes things faster.